### ISSUE TITLE
"Most Played" data incorrect

### ISSUE THREAD

**Issue by blackparadox** [felixkmh/StartPage-for-Playnite#147]
- **StartPage Version:** 2.6.0
- **Playnite Version**: 9.18

**TLDR:** The "Most Played" widget is displaying the same game for all time periods except "All Time".

In my current configuration, I have Borderlands 3 displaying as the most played in the "Last Month", "Last Year" and "Last 5 Years", and Final Fantasy XIV in "All Time".

But what I would expect to see is;

- Last Month: Borderlands 3 (last played 13/06/2022, total playtime 35 hours)
- Last Year: Assassin's Creed: Valhalla (last played 08/04/2022, total playtime 128h)
- Last 5 Years: The Witcher 3: Wild Hunt (last played 09/02/2020, total playtime 165h)
- All Time: Final Fantasy XIV (last played 29/05/2017, total playtime 748h)

This was working before the update that allowed us to choose the displayed time periods, and stopped working as soon as that update was applied (before changing the display settings). 

I've tried multiple combinations since but it always displays Borderlands 3 for everything except "All Time".

If I tag Borderlands 3 as "[SPG] Most Played Hadden" and restart Playnite, the bug persists but with the next most played game of the month appearing in all panels (except "All Time") - in my example this is Just Cause 4 (last played 16/05/2022, total playtime 20h) - removing this tag puts Borderlands 3 back in them all.

**Comment by felixkmh** [comment]
Couldn't you just use a game shelve sorted by playtime for that?
